backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToRemoteStatusToRemoteUnitTestSuite.java

```diff
@@ -28,7 +29,7 @@
   private final IGitCoreCommit coreRemoteBranchCommit = PowerMockito.mock(IGitCoreCommit.class);
 
   @SneakyThrows
-  private SyncToRemoteStatus invokeDeriveSyncToRemoteStatus(IGitCoreLocalBranchSnapshot coreLocalBranch) {
+  private RelationToRemote invokeDeriveSyncToRemoteStatus(IGitCoreLocalBranchSnapshot coreLocalBranch) {
     return Whitebox.invokeMethod(aux(), "deriveSyncToRemoteStatus", coreLocalBranch);
```

Comment:
Hmm is this method still called `deriveSyncToRemoteStatus` even though it returns a `RelationToRemote`?

Reply:
Tip: Ctrl+Shift+Alt+N to locate any **symbol**, including class, method etc.

Reply:
(while Ctrl+N just looks for any **class**)